tasks/fix-cat2.yml

```
@@ -1524,6 +1524,7 @@
       regexp: (?i)^#?compression
       line: Compression no
       validate: sshd -t -f %s
+  ignore_errors: yes
```

Why ignore errors for this task?